### PR TITLE
Canva type picker unit tests

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Canva type picker unit tests

## Motivation and Context
Canva integration

## How Has This Been Tested?
Tests pass
[stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: test/unit/components/canva-type-picker/ctr-canva-type-picker.spec.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  it("should exist",function(){
+    expect($scope).to.be.truely;
+    expect($scope.designType).to.equal("Flyer");
+
+    expect($scope.ok).to.be.a("function");
+    expect($scope.dismiss).to.be.a("function");
+  });
+
+  describe("ok:", function() {
+    it("should close modal and resolve default desing type",function(){
```
Comment (alex-deaconu) [inline test/unit/components/canva-type-picker/ctr-canva-type-picker.spec.js:34]: @ezequielc typo here `desing`

File: test/unit/components/canva-type-picker/ctr-canva-type-picker.spec.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      $scope.ok();
+      $scope.$digest();
+      $modalInstance.close.should.have.been.calledWith("Flyer");
+    });
+
+    it("should close modal and resolve updated desing type",function(){
```
Comment (alex-deaconu) [inline test/unit/components/canva-type-picker/ctr-canva-type-picker.spec.js:40]: And here.

**Review by alex-deaconu — APPROVED** [review]
Looks good 👍

**Comment by ezequielc** [comment]
Thanks @alex-deaconu, I updated the typos. Will merge once build finishes.

**Pull request merged**